pkg/machines/components/storagePools/storagePoolOverviewTab.jsx

```diff
@@ -33,6 +33,12 @@ export const StoragePoolOverviewTab = ({ storagePool, actionError, actionErrorDe
     return (
         <div className='ct-form-layout'>
             { actionError && <ExpandableNotification type='warning' text={actionError} textId={`${idPrefix}-error`} detail={actionErrorDetail} onDismiss={onActionErrorDismiss} /> }
+            { storagePool.source && storagePool.source.host && <React.Fragment>
+                <label className='control-label' htmlFor={`${idPrefix}-host`}> {_("Host")} </label>
+                <div id={`${idPrefix}-host`}>
```

Comment:
Can this please get a corresponding test? Or don't we have a host name in our test env? (like "localhost"?)

Reply:
The last commit adds such a test.